### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by NotReshab** [NotSaksh/Discord-Server-Cloner#3] [General]
Hello , the cloner has a bug  , it don't clone server messages

**Comment by SakshL** [comment]
will be fixed soon

**Reply by TheKiller000** [reply]
Plaese Fixed

**Comment by SakshL** [comment]
> Hello , the cloner has a bug , it don't clone server messages

will be fixe dsoon

**Comment by TheKiller000** [comment]
Hello, this file does not have start.bat, please fix it
Thanks
